src/components/Button/Button.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Button = ({ nameButton, action }) => (
```

Comment:
краще деструктуризацію проводити з тими ж назвами пропсів, які передаєш **IMHO** задля підвищення читабельності.
```suggestion
export const Button = ({ nameButton, onClick}) => (
```

І ще одне: в тебе назва змінної названа з дієслова. З дієслів, як правило, починаються функції
```suggestion
export const Button = ({ buttonName, action }) => (
```

Reply:
Согласен!